cmd/titond/main.go

```diff
@@ -10,6 +10,7 @@ import (
 	"github.com/tokamak-network/tokamak-titond-backend/pkg/http"
 	"github.com/tokamak-network/tokamak-titond-backend/pkg/kubernetes"
 	"github.com/tokamak-network/tokamak-titond-backend/pkg/services"
+	apptest "github.com/tokamak-network/tokamak-titond-backend/test"
```

Comment:
I think we can use `test` as a package name now. It is small thing. Thank you! 😍

Reply:
I updated